install/cluster-operator/050-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -62,7 +62,7 @@ spec:
         - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
           value: strimzi/operator:latest
         - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
-          value: strimzi/kafka-bridge:latest
+          value: strimzi/kafka-bridge:0.15.0
```

Comment:
why do you think it's better having the master using latest released bridge version and not the one not released yet ("latest"). What happens if we are developing a feature in the operator which needs the "latest" bridge image while it's pointing to use a specifc released version (i.e. 0.15.0 in this case). I know that during development you can just change this to "latest" (instead of 0.15.0) but isn't it better having "under development" stuff used together?

Reply:
I was thinking about it yesterday. But I think that it would not work well. You would be getting into a stituation where we have working master but when we do the operator release the bridge stops working because we would basically downgrade the bridge to older version.

I think that instead it is better to develop the new bridge features, where needed test them with operators by modifying the YAML files and only when it is ready and released you would open a PR to update operators and the bridge version. At the end that is how we work with the Strimzi OAuth library, with KAfka it self etc.

Reply:
Comparing bridge development and related integration into operator to oauth, kafka etc makes sense, so I am fine with it.

Reply:
Great, thanks.